core/src/main/resources/reference.conf

```diff
@@ -535,6 +535,41 @@ datastax-java-driver {
     #   change.
     # Overridable in a profile: no
     warn-on-init-error = true
+
+
+    advanced-shard-awareness {
+      # Whether to use advanced shard awareness when trying to open new connections.
+      #
+      # Having this enabled makes sense only for ScyllaDB clusters.
+      # Results in smaller connection storms in multi-client settings.
+      # If set to false the driver will not attempt to use this feature.
```

Comment:
Can you please change it to something like this: `if set to false the driver will not enforce local port on connection socket, counting on the fact that it eventually going to get connection to the shard it wants.`

Reply:
Added some extra text and linked blogpost about adv shard awareness